nsqadmin/logger.go

```diff
@@ -1,5 +1,58 @@
 package nsqadmin
 
+import (
+	"fmt"
+	"strings"
+)
+
 type Logger interface {
 	Output(maxdepth int, s string) error
 }
+
+const (
+	LOG_DEBUG = 1
+	LOG_INFO  = 2
+	LOG_WARN  = 3
+	LOG_ERROR = 4
+	LOG_FATAL = 5
+)
+
+func (n *NSQAdmin) logLevelFromString(level string) int {
```

Comment:
Agreed about sharing across applications, but was thinking it would be OK if @antihax did not want to do that refactoring in this pull request. We could have the cmdline flags and http interfaces finalized in this PR, and I could do internal refactoring in a follow-up (and we could "bikeshed" as necessary).

I normally don't entertain promises to clean up in a later pull-request, but since it's me ¯\\_(ツ)_/¯

Or, instead of directly merging this, I could include and then build on it in a separate PR.